setup/CMakeLists.txt

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.15)
+
+project(dependencies VERSION 1 LANGUAGES C CXX)
+
+include(ExternalProject)
+include(CMakePrintHelpers)
+
+option(USE_SUDO       "Use sudo when installing" OFF)
+
+if(USE_SUDO)
+  set(SUDO_CMD "sudo")
```

Comment:
sudo -E
isn't this better if we are to capture any existing proxy settings ?

Reply:
Done